### ISSUE TITLE
Intermittent CI failure in step `Install Carrier`. Deployment troubles, timeout in waiting for completion.

### ISSUE THREAD

**Issue by andreas-kupries** [SUSE/carrier#34]
```
Deploying Quarks
⣾ Waiting for resource  to be running in quarks ... 💤   ...
Couldn't install carrier: failed waiting Quarks deployment to come up: no pods in quarks with selector 
Error: Process completed with exit code 1.
```

**Comment by andreas-kupries** [comment]
Can happen fir gitea as well. Might be too short a wait period, i.e. when GH is loaded and slower in performing deployment.

**Comment by andreas-kupries** [comment]
Actually the cli is too fast, starting to wait for a pod to be running without waiting for the pod to exist.

See PR #47 (merged) and #52 (pending) for fixes.
